lib/src/tooltip_widget.dart

```diff
@@ -589,3 +590,18 @@ class _Arrow extends CustomPainter {
         oldDelegate.strokeWidth != strokeWidth;
   }
 }
+
+class ToolTipSlideTransition extends AnimatedWidget {
```

Comment:
This file is getting very big now, can you please saparate code in multiple files.